utilities/game.py

```diff
@@ -24,7 +25,7 @@ class GamePiece(Enum):
     NONE = auto()
 
 
-class Rows(Enum):
+class Rows(IntEnum):
```

Comment:
Let's avoid adding more IntEnums. If you need an int just use the enum member `.value`.